app/models/record.rb

```diff
@@ -3,4 +3,18 @@ class Record < ApplicationRecord
   belongs_to :register
   scope :current, -> { where("data->> 'end-date' is null") }
   scope :archived, -> { where("data->> 'end-date' is not null") }
+  scope :status, lambda { |status|
```

Comment:
Does this need to be a lambda?

Reply:
this style is preferred by Rubocop for multi-line functions. http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/Lambda